### ISSUE TITLE
Crash of the application on startup

### ISSUE THREAD

**Issue by Porkepix** [readrops/Readrops#51]
After installing from F-droid, I managed to run the application one time, got the welcome screen where I selected add a local RSS. The application crashed. Since then, I can't even open it anymore, it crashes directly on startup.

Issue experienced on Android 6.0.1 with standard manufacturer ROM (Sony).

Here's the `adb logcat -v threadtime` after a crash:

```
02-14 02:06:16.110  3822  3822 I Timeline: Timeline: Activity_launch_request id:com.readrops.app time:172667849
02-14 02:06:16.111   925  4438 I ActivityManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.readrops.app/.activities.SplashActivity bnds=[192,742][360,938]} from uid 10110 on display 0
02-14 02:06:16.156   925  3837 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@1aa0df0
02-14 02:06:16.156   925  3837 I ActivityManager: Start proc 4801:com.readrops.app/u0a471 for activity com.readrops.app/.activities.SplashActivity
02-14 02:06:16.248  4801  4801 W System  : ClassLoader referenced unknown path: /data/app/com.readrops.app-1/lib/arm
02-14 02:06:16.545  4801  4829 D OpenGLRenderer: Use EGL_SWAP_BEHAVIOR_PRESERVED: true
02-14 02:06:16.574  4801  4829 I Adreno-EGL: <qeglDrvAPI_eglInitialize:379>: EGL 1.4 QUALCOMM build: Nondeterministic_AU_msm8974_LA.BF.1.1.3_RB1__release_AU (I741a3d36ca)
02-14 02:06:16.574  4801  4829 I Adreno-EGL: OpenGL ES Shader Compiler Version: E031.29.00.00
02-14 02:06:16.574  4801  4829 I Adreno-EGL: Build Date: 04/04/16 Mon
02-14 02:06:16.574  4801  4829 I Adreno-EGL: Local Branch: mybranch19053788
02-14 02:06:16.574  4801  4829 I Adreno-EGL: Remote Branch: quic/LA.BF.1.1.3_rb1.12
02-14 02:06:16.574  4801  4829 I Adreno-EGL: Local Patches: NONE
02-14 02:06:16.574  4801  4829 I Adreno-EGL: Reconstruct Branch: NOTHING
02-14 02:06:16.577  4801  4829 I OpenGLRenderer: Initialized EGL, version 1.4
02-14 02:06:16.618  4801  4801 I Timeline: Timeline: Activity_launch_request id:com.readrops.app time:172668356
02-14 02:06:16.618   925  3317 I ActivityManager: START u0 {cmp=com.readrops.app/.activities.MainActivity} from uid 10471 on display 0
02-14 02:06:16.627   925  3848 W InputMethodManagerService: Focus gain on non-focused client com.android.internal.view.IInputMethodClient$Stub$Proxy@bbff487 (uid=10471 pid=4801)
02-14 02:06:16.669   925   989 I Timeline: Timeline: Activity_windows_visible id: ActivityRecord{116b4a u0 com.readrops.app/.activities.SplashActivity t6284 f} time:172668408
02-14 02:06:16.918  4801  4801 I art     : Rejecting re-init on previously-failed class java.lang.Class<i.l0.h.b$b>
02-14 02:06:16.918  4801  4801 I art     : Rejecting re-init on previously-failed class java.lang.Class<i.l0.h.b$b>
02-14 02:06:17.030  4801  4801 W ResourceType: Failure getting entry for 0x7f08006f (t=7 e=111) (error -75)
02-14 02:06:17.030  4801  4801 D AndroidRuntime: Shutting down VM
02-14 02:06:17.033  4801  4801 E AndroidRuntime: FATAL EXCEPTION: main
02-14 02:06:17.033  4801  4801 E AndroidRuntime: Process: com.readrops.app, PID: 4801
02-14 02:06:17.033  4801  4801 E AndroidRuntime: android.content.res.Resources$NotFoundException: Resource ID #0x7f08006f
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at android.content.res.Resources.getValue(Resources.java:1358)
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at androidx.appcompat.widget.i0.e()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at androidx.appcompat.widget.i0.a()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at androidx.appcompat.widget.i0.a()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at d.a.k.a.a.c()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at androidx.appcompat.widget.m.a()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at androidx.appcompat.widget.AppCompatImageView.setImageResource()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at e.c.d.p.d.a()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at e.c.e.k.c.a()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at e.c.d.b.a()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at com.readrops.app.utils.b.c()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at com.readrops.app.utils.b.a()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at com.readrops.app.activities.MainActivity.a()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at com.readrops.app.activities.s.a(lambda)
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at androidx.lifecycle.LiveData.b()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at androidx.lifecycle.LiveData.a()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at androidx.lifecycle.LiveData.b()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at androidx.lifecycle.LiveData$a.run()
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:739)
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:234)
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:5526)
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
02-14 02:06:17.033  4801  4801 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

**Comment by Shinokuni** [comment]
Hello, thanks for the bug report, I will investigate. 

Unfortunately, I didn't really test Readrops on older API levels (below nougat/oreo), otherwise I would have fixed this kind of issue quicker.

**Comment by Shinokuni** [comment]
For a reason I ignore, the drawer header background was only accessible for API >= 24. For APIs below, it leads to the drawer to crash.

It is fixed in a960f59e2f86bafff4f5d556507c20e021a693f2.
